web/src/app/schedules/ScheduleRuleEditDialog.js

```diff
@@ -57,30 +59,34 @@ export default class ScheduleRuleEditDialog extends React.Component {
         variables={{ id: this.props.scheduleID, tgt: this.props.target }}
         noPoll
         fetchPolicy='network-only'
-        render={({ data }) => this.renderMutation(data.schedule.target)}
+        render={({ data }) =>
+          this.renderMutation(data.schedule.target, data.schedule.timeZone)
+        }
       />
     )
   }
 
-  renderMutation(data) {
+  renderMutation(data, zone) {
     return (
       <Mutation mutation={mutation} onCompleted={this.props.onClose}>
-        {(commit, status) => this.renderDialog(data, commit, status)}
+        {(commit, status) => this.renderDialog(data, commit, status, zone)}
       </Mutation>
     )
   }
 
-  renderDialog(data, commit, status) {
+  renderDialog(data, commit, status, zone) {
     const defaults = {
       targetID: this.props.target.id,
-      rules: data.rules.map(r =>
-        pick(r, ['id', 'start', 'end', 'weekdayFilter']),
-      ),
+      rules: data.rules.map(r => ({
+        ..._.pick(r, ['id', 'weekdayFilter']),
```

Comment:
Could you explain how the spread operator works before a lodash func (`..._.pick`)? I haven't seen it used in this context before so it's a bit confusing to me